ssc_interface_wrapper_ros2/launch/ssc_params.yaml

```diff
@@ -0,0 +1,15 @@
+# SSC assumes file names that don't start with / are in the same directory as the launch file
```

Comment:
Do you know enough now to make this comment clearer?

Reply:
Yes, this comment can be removed. The params file is taking the path as it is given, which would be as expected